DUI3-DX/Connectors/Revit/Speckle.Connectors.Revit2023/Speckle.Connectors.Revit2023.csproj

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <TargetFramework>net48</TargetFramework>
     <PlatformTarget>x64</PlatformTarget>
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Was this breaking something? My assumption was we always wanted to target win-x64 in Revit.

Reply:
MSIL mismatch because of revit AMD64

Reply:
CI was passing before damn it